classes/components/forms/FieldContrastColorPicker.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @file classes/components/form/FieldContrastColorPicker.php
+ *
+ * Copyright (c) 2025 Simon Fraser University
+ * Copyright (c) 2025 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class FieldContrastColorPicker
+ *
+ * @ingroup classes_controllers_form
+ *
+ * @brief A color picker field with automatic contrast color suggestions.
+ */
+
+namespace PKP\components\forms;
+
+class FieldContrastColorPicker extends Field
+{
+    /** @copydoc Field::$component */
+    public string $component = 'field-contrast-color-picker';
+
+    /** The selected contrast color */
+    public string $contrastColor = '';
+
+    /**
+     * @copydoc Field::getConfig()
+     */
+    public function getConfig(): array
+    {
+        $config = parent::getConfig();
+        $config['contrastColor'] = $this->contrastColor;
+
+        return $config;
```

Comment:
I think this syntax looks nicer, but it's just my opinion, no need to change ¯\_(ツ)_/¯

```suggestion
        return parent::getConfig() + ['contrastColor' => $this->contrastColor];
```

Reply:
I like that too. It's shorter and more direct haha.